app/assets/javascripts/discourse/app/widgets/search-menu.js

```diff
@@ -198,6 +198,7 @@ export default createWidget("search-menu", {
   defaultState(attrs) {
     return {
       inTopicContext: attrs.inTopicContext,
+      inPMInboxContext: this.search?.searchContext?.type === "private_messages",
```

Comment:
I may be wrong but couldn't you use the existing `this.searchContext()` method here as we do below in `fullSearchUrl`?

Reply:
Good thinking, but we actually can't use the existing `this.searchContext()` because that one is only set based on the state, and in `defaultState` we don't have a state yet. (The naming here is not ideal, I normally would opt to rename things, but this widget is extended/overriden in multiple theme components, and renaming would trigger multiple other changes.)

Reply:
Yeah that's cool wasn't sure if it would be possible :+1: